CONTRIBUTING.md

```diff
@@ -1,91 +1,101 @@
+ - If that is not possible either, provide some info (numbers, screenshots, testing procedure) on how you tested your stuff
+ - If all is lost, state why you have not been able to do the above.
 
-See also [bisq-network/style#9](https://github.com/bisq-network/style/issues/9).
+## Create a pull request
+- Pull requests have to be focused on a single change. Do not mix unrelated stuff.
+- Use meaningful title and description
+  - provide some context, how did stuff work before, how does it work now, why is that better
+  - are there some introduced/mitigated risks, maybe for backwards compatibility
+  - how you tested your stuff (so we can do so as well if we feel so)
+  - and of course, reference issues (if applicable) with the use of [keywords](https://help.github.com/en/github/managing-your-work-on-github/closing-issues-using-keywords)
+- Wait for your pull request to be reviewed
 
-### Sign your commits with GPG
-
-See https://github.com/blog/2144-gpg-signature-verification for background and
-https://help.github.com/articles/signing-commits-with-gpg/ for instructions.
-
-### Use an editor that supports Editorconfig
-
-The [.editorconfig](.editorconfig) settings in this repository ensure consistent management of whitespace, line endings and more. Most modern editors support it natively or with plugin. See http://editorconfig.org for details. See also [bisq-network/style#10](https://github.com/bisq-network/style/issues/10).
+## Pull request review prodecure
+Bisq follows the review workflow established by the Bitcoin Core project. The following is adapted from the [Bitcoin Core contributor documentation](https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md#peer-review): 
+ - `ACK` means "I have tested the code and I agree it should be merged";
+ - `NACK` means "I disagree this should be merged", and must be accompanied by sound technical justification. NACKs without accompanying reasoning may be disregarded;
+ - `utACK` means "I have not tested the code, but I have reviewed it and it looks OK, I agree it can be merged";
+ - `Concept ACK` means "I agree in the general principle of this pull request";
+ - `Nit` refers to trivial, often non-blocking issues.
 
-### Keep the git history clean
+In case you receive a NACK and/or `change requested`
+ - react within 30 days, or else, your pull request will be rejected automatically
+ - **re-request a review** after adressing a change request (because we almost always only look at "review required" pull request because of resource constraints)
 
-It's very important to keep the git history clear, light and easily browsable. This means contributors must make sure their pull requests include only meaningful commits (if they are redundant or were added after a review, they should be removed) and _no merge commits_.
+## Compensation
 
-### Additional style guidelines
+Your pull request got merged? great, congrats and thanks for making Bisq more better.
```

Comment:
```suggestion
Your pull request got merged? Great! Congrats on your first contribution and thanks for making Bisq better.
```

Reply:
I rather not have the "on your first contribution" phrase in there because the section this text can be found is applicable for each and every pull request.

Thoughts?